inputs/select.vue

```diff
@@ -58,19 +61,23 @@
   <ui-select
     v-if="hasOptions"
     :name="name"
-    :value="safeData"
+    :value="value"
+    :keys="keys"
     :options="options"
     :multiple="args.multiple"
     :hasSearch="args.search"
+    :list="args.list"
     :label="label"
     :floatingLabel="true"
     :help="args.help"
     :error="errorMessage"
     :invalid="isInvalid"
     iconPosition="right"
-    @input="update">
+    @input="handleInput"
+  >
     <attached-button slot="icon" :name="name" :data="data" :schema="schema" :args="args" @disable="disableInput" @enable="enableInput"></attached-button>
   </ui-select>
+  <!-- todo: there's a "no-results" slot in ui-select, should we maybe use that? -->
```

Comment:
probably!

Reply:
i made a separate ticket for this (#1140), seems nice to have and I wanna learn about how to use vue slots, but it's not something we need to make this PR shippable